appveyor.yml

```diff
@@ -24,9 +24,16 @@ before_build:
 
 # scripts to run after build
 # ...testing
+# Note we do a ping, do save time if the test server is down.
+# Most of the rest of this is to avoid running tests on vendored dirs.
 after_build:
+  - keybase ping
   - cd ..
-  - go test -timeout 30m ./...
+  - go list ./... | find /V "vendor" > testlist.txt
```

Comment:
Well this didn't work on appveyor anyway - what was yours?
On Nov 23, 2015 6:00 PM, "Maxwell Krohn" <notifications@github.com> wrote:

> In appveyor.yml
> <https://github.com/keybase/client/pull/1392#discussion_r45689329>:
>
> >    - cd ..
> > -  - go test -timeout 30m ./...
> > +  - go list ./... | find /V "vendor" > testlist.txt
>
> Fine by me. I'll back out my change.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/keybase/client/pull/1392/files#r45689329>.
>